sdk/ml/azure-ai-ml/azure/ai/ml/_schema/automl/image_vertical/image_object_detection.py

```diff
@@ -8,7 +8,7 @@
 
 from marshmallow import fields, post_load
 
-from azure.ai.ml._restclient.v2022_10_01_preview.models import (
+from azure.ai.ml._restclient.v2022_12_01_preview.models import (
```

Comment:
Maybe automl can centralize these imports in some file, so its easier to change next time

Reply:
This is great suggestion. Will prevent changes across so many files just to use next API version

Reply:
Created a WI: https://msdata.visualstudio.com/Vienna/_workitems/edit/2266774